src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$cell-size: 100px;
+$gap-size: 1px;
```

Comment:
Consider using more descriptive variable names, like $calendar-gap-size, for better readability.